src/web/app/views/beambox/Photo-Edit-Panel.tsx

```diff
@@ -280,8 +278,8 @@ class PhotoEditPanel extends React.Component<Props, State> {
         autoCropArea: 1,
         zoomable: false,
         viewMode: 0,
-        targetWidth: image.width,
-        targetHeight: image.height,
+        // targetWidth: image.width,
+        // targetHeight: image.height,
```

Comment:
@dkdkwizard 
https://www.npmjs.com/package/cropperjs#options
`targetWidth` and `targetHeight` are not accepted. Not sure if they are really needed.

Reply:
They are not needed.